_layouts/docs.html

```diff
@@ -423,6 +445,7 @@
   <script src="/js/highlight.pack.js"></script>
   <script>hljs.initHighlightingOnLoad();</script>
   <script src="/js/alljs.js"></script>
+  <script src="/js/bootstrap.min.js"></script>
```

Comment:
I am pretty sure we already have Bootstrap JS included? We are trying really hard not to have to include new Javascript, since we already have a 1MB monolithic file.